.github/workflows/main.yml

```diff
@@ -41,50 +70,104 @@ jobs:
         run: |
           wget -q https://github.com/pyodide/pyodide/releases/download/${{ matrix.pyodide-version }}/pyodide-build-${{ matrix.pyodide-version }}.tar.bz2
           tar xjf pyodide-build-${{ matrix.pyodide-version }}.tar.bz2
-          mv pyodide dist/
+          mv pyodide pyodide-dist/
+  make_test_matrix:
+    outputs:
+      matrix: ${{ steps.filter.outputs.matrix }}
+      pyodide_versions: ${{ steps.filter.outputs.pyodide_versions }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Write py file
+        run: |
+          cat << 'EOF' > filtermatrix.py
+          import yaml,json
+
+          matrix=yaml.safe_load("""
+            os: [ubuntu-latest]
+            pyodide-version: [0.21.0a3,0.21.0]
+            test-config: [
+              {runner: selenium, runtime: firefox, firefox-version: latest, geckodriver-version: latest },
+              {runner: selenium, runtime: chrome, chrome-version: latest, chromedriver-version: latest },
+              {runner: selenium, runtime: node, node-version: 14},
+              {runner: selenium, runtime: node, node-version: 16},
+              {runner: selenium, runtime: node, node-version: 18},
+              {runner: selenium, runtime: firefox-no-host, firefox-version: latest, geckodriver-version: latest },
+              {
+                runner: selenium, runtime: chrome firefox,
+                firefox-version: latest, geckodriver-version: latest,
+                chrome-version: latest, chromedriver-version: latest,
+              },
+              {runner: selenium, runtime: host},
+              # playwright browser versions are pinned to playwright version
+              {runner: playwright, runtime: firefox, playwright-version: 1.22.0, node-version: 18},
+              {runner: playwright, runtime: chrome, playwright-version: 1.22.0, node-version: 18},
+            ]
+            include:
+              - os: macos-latest
+                pyodide-version: 0.21.0
+                test-config: {runner: selenium, runtime: safari}
+              - os: macos-latest
+                pyodide-version: 0.21.0
+                test-config: {runner: selenium, runtime: safari, refresh: 1 }
```

Comment:
It makes sense to test the `STANDALONE_REFRESH` in this repo, but it doesn't really make sense to expose it as a separate Pyodide version for other packages IMO. They should rather specify this env variable if they need to.